sdk/devcenter/azure-developer-devcenter/src/test/java/com/azure/developer/devcenter/custom/CreatesADevBoxTests.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.developer.devcenter.custom;
+
+import com.azure.core.util.polling.LongRunningOperationStatus;
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.developer.devcenter.models.DevBox;
+import com.azure.developer.devcenter.models.DevCenterOperationDetails;
+import com.azure.developer.devcenter.DevCenterClientTestBase;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+@Disabled
+public final class CreatesADevBoxTests extends DevCenterClientTestBase {
+    @Test
+    @Disabled
```

Comment:
What's the `custom` package for? Why tests here stay disabled?

Reply:
For both dev box and environment, I have a customization that changes the number of parameters in the create methods.  For the model and client generation I can customize removing the `@generated` on top and then making my changes, but for tests and samples there is no really good way to customize the generation, so I just copied whatever was generated to the `custom` folder and updated the method here.

Reply:
The generated tests and samples are good to ensure it compiles, that is why here is `disabled`. But for every single api I have my own tests in [DevBoxTests.java](https://github.com/Azure/azure-sdk-for-java/blob/drielene/generate-with-models/sdk/devcenter/azure-developer-devcenter/src/test/java/com/azure/developer/devcenter/DevBoxTests.java), [EnvironmentTests.java](https://github.com/Azure/azure-sdk-for-java/blob/drielene/generate-with-models/sdk/devcenter/azure-developer-devcenter/src/test/java/com/azure/developer/devcenter/EnvironmentTests.java) and [DevCenterTests.java](https://github.com/Azure/azure-sdk-for-java/blob/drielene/generate-with-models/sdk/devcenter/azure-developer-devcenter/src/test/java/com/azure/developer/devcenter/DevCenterTests.java),

Reply:
we can see the records for each test in [here](https://github.com/Azure/azure-sdk-assets/tree/java/devcenter/azure-developer-devcenter_eae6bce1e1/java/sdk/devcenter/azure-developer-devcenter/src/test/resources/session-records)

Reply:
The generated test is just for reference (that's the reason they are put to `generated` package). You are free to delete any test/sample that no longer applies (that includes the case the you've customized the API and the generated test/sample calling the origin API may not work or not compile).